xml_inscriptions/bib_only/MA.Bos.MFA.G.21.1843.xml

```diff
@@ -62,17 +133,33 @@
       <revisionDesc>
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-07-13" who="Jean Wanlass">added metadata, transcription,
+            translation</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.21.1843.jpg"/>
+      </surface>
+   </facsimile>
+   
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+            <ab>ϘΥΦΒΓΣΔΑΣΑΝΒΘΒΝ ΒΙΒΡΑΚΓΕΑΜ</ab></div>
```

Comment:
Each line of transcription needs to be preceded by an `<lb>`; transcription needs to include accents, to the extent possible in the dialect you're dealing with

Reply:
I fixed